server/src/main/java/info/doseamigos/authusers/AuthUser.java

```diff
@@ -5,22 +5,25 @@
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import org.apache.commons.lang3.builder.ToStringBuilder;
 
+import java.math.BigInteger;
+
 import static java.util.Objects.requireNonNull;
 
 /**
  * POJO for users that authenticate with Google.
  */
 public class AuthUser {
 
-    private long authUserId;
+    private Long authUserId;
     private AmigoUser amigoUser;
     private String email;
+    private BigInteger googleRef;
```

Comment:
Google's id is too big for a long, so I'm using BigInteger, and for the DB, I'm storing the string representation.